packages/cloud-cognitive/src/components/Tearsheet/_tearsheet.scss

```diff
@@ -216,6 +216,18 @@
 
   .#{$block-class}.#{$block-class}--wide .#{$block-class}__main {
     background: $ui-02;
+
```

Comment:
@dcwarwick Some of the input components have a 'light' version. Should we cater for that too?

Reply:
@lee-chase Carbon kills the 'light' prop in modals. However, it's easy enough to restore the 'light' behaviour as well as the background colour, so I've updated to do that.